wazo_acceptance/steps/sip_endpoint.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from behave import when
+
+
+@when(u'I set the following options in line "{exten}@{exten_context}"')
+def when_i_update_lines_with_infos(context, exten, exten_context):
```

Comment:
fix name of the method
also remove the `u` before string
And since the options are not related to line, but to the endpoint, I propose:
'I set the following options in endpoint_sip "{exten}@{exten_context}"'